### DISCUSSION TITLE
restart/reboot directus server

### DISCUSSION THREAD

**Discussion by dieriba** [directus/directus#16150] [Archive]
After a sudo reboot on my digital ocean droplet, my directus server is not joinable anymore at his url which is  "api.mira.ohmytag.fr", i get 502 error response each time i try to go this url.
i did pm2 list but directus is not here anymore and when I try the command "npx directus start" they say that port is already in use.

### To Reproduce

try to join the url api.mira.ohmytag.fr

### Errors Shown

502 error in browser.

### What version of Directus are you using?

9.18.1

### What version of Node.js are you using?

16.x.x

### What database are you using?

Sql

### What browser are you using?

Edge

### How are you deploying Directus?

Docker

**Comment by br41nslug** [comment]
Any errors on the server side?
How were you running Directus before the reboot? You mention 3 very different ways of deploying in this issue which one is it? `Docker`, `pm2` or `npx directus start`

**Comment by dieriba** [comment]
Hey nox directus start works but once I try pm2 restart i have this error.
I did not try docker

**Comment by br41nslug** [comment]
> Hey npx directus start works but once I try pm2 restart i have this error.

Those are 2 different ways of starting directus. You should either start it with `npx directus start` or `pm2 start ....`

**Comment by dieriba** [comment]
Yes but we made pm2 restart and it does not work, precisely when we update
or env and aff oauth params and then do the command pm2 restart we get this
error while if we add nothing to the env the commands works

**Comment by dieriba** [comment]
And the server is currently could you check it from your side to see what
happened or is happening
Le mar. 25 oct. 2022 à 11:32, Dieriba toure ***@***.***> a
écrit :

**Reply by rijkvanzanten** [reply]
We can't. You're self hosting directus, so we don't have any access or insight and can't really offer any support. Consider using Directus Cloud if you don't want to deal with hosting problems 👍🏻
